src/game/level.rs

```diff
@@ -0,0 +1,31 @@
+//! Setup the level
+
+use bevy::prelude::*;
+
+use super::physics::{PhysicalTransform, Velocity};
+use crate::screen::Screen;
+
+#[derive(Debug, Event)]
+pub(crate) struct SpawnLevel;
+
+pub(super) fn plugin(app: &mut App) {
+    app.observe(spawn_level);
+}
+
+fn spawn_level(
+    _trigger: Trigger<SpawnLevel>,
+    mut commands: Commands,
+    asset_server: Res<AssetServer>,
+) {
+    commands.spawn((
+        Name::new("Player"),
+        SpriteBundle {
+            texture: asset_server.load("ducky.png"),
```

Comment:
Do we want to add a loading stage for game assets later?

Reply:
Yep, that would be nice. #71